plugins/woocommerce-admin/client/guided-tours/report-date-tour.tsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import { TourKit, TourKitTypes } from '@woocommerce/components';
+import { __ } from '@wordpress/i18n';
+import { OPTIONS_STORE_NAME } from '@woocommerce/data';
+import {
+	createElement,
+	createInterpolateElement,
+	useState,
+} from '@wordpress/element';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { getAdminLink } from '@woocommerce/settings';
+
+/**
+ * Internal dependencies
+ */
+import './report-date-tour.scss';
+
+export const ReportDateTour: React.FC< {
+	optionName: string;
+	headingText: string;
+} > = ( { optionName, headingText } ) => {
+	const [ isDismissed, setIsDismissed ] = useState( false );
+	const { updateOptions } = useDispatch( OPTIONS_STORE_NAME );
+
+	const { hasShownTour, isResolving } = useSelect( ( select ) => {
+		const { getOption, hasFinishedResolution } =
+			select( OPTIONS_STORE_NAME );
+
+		return {
+			hasShownTour: getOption( optionName ) === 'yes',
```

Comment:
Would it also be worth checking if the `woocommerce_date_type` is actually set to `date_paid` or maybe still `undefined` (meaning it has not been set yet)?

Mostly wondering as @nathanss PR that adds the `woocommerce_date_type` option has already been merged before the code freeze. So this PR won't make it in until the next release (unless we make a code freeze exception).
This could mean that users have already changed that option themselves and were already aware of that.

Reply:
I would leave it like this. I might be wrong but I don't expect many users to change this option before the guided tour, since it's kind of hidden in the analytics settings. @octaedro what do you think?

Reply:
Maybe we can confirm this with @pmcpinto or @AnnaMag

Reply:
@octaedro it would be helpful to include this alongside other change, but I'm afraid it does not qualify for a code freeze exception. Do you think differently? 

If we aim for 7.6 with this change, I'd agree with @louwie17's suggestion to be on a safe side as we have seen cases of users manipulating hidden and apparently low-traffic options.

Reply:
Yeah I am afraid it won't qualify for a code freeze exception either.
The change should be relatively straight forward, so I don't think it will hurt adding it in, even if the impact will be low.

Reply:
I will implement this today